pysyncobj/tcp_connection.py

```diff
@@ -232,12 +234,17 @@ def __processParseMessage(self):
         data = self.__readBuffer[4:4 + l]
         try:
             if self.encryptor:
+                dataTimestamp = self.encryptor.extract_timestamp(data)
+                assert dataTimestamp >= self.recvLastTimestamp
+                self.recvLastTimestamp = dataTimestamp
+                # Unfortunately we can't get a timestamp and data in one go
                 data = self.encryptor.decrypt(data)
             message = pickle.loads(zlib.decompress(data))
             if self.recvRandKey:
                 randKey, message = message
                 assert randKey == self.recvRandKey
         except:
+            # Why no logging of security errors?
```

Comment:
You can add logging

Reply:
Ha, of course. I see PySyncObj already has `logging` imported so printing something like "invalid message received from $ip, closing connection" should be not hard.  I'll do a separate PR after I figure exactly how.

Reply:
Hm, do you think it should be `logger.warning` or more like `logger.info`? On private network it feels like warning because it means I have broken app somewhere, but on internet it feels wrong because I receive lot of junk, so its more like `info` or `debug`. Er, is it good idea to write anything to log just because somebody from internet connected?

wow... you are fast! Thank you.

Reply:
Probably that's the reason it was missing in log ) Probably we can leave it as is for now..